external-crates/move/crates/move-analyzer/src/bin/move-analyzer.rs

```diff
@@ -216,8 +216,13 @@ fn main() {
                             },
                         }
                     },
-                    Err(error) =>
-                        eprintln!("symbolicator message error: {:?}", error),
+                    Err(error) => {
```

Comment:
This is not directly related to the fix but while debugging this with the help of an external developer I realized that it would simplify things if we had to go through a similar exercise in the future.